### PR TITLE
Services Docs update for arm64

### PR DESCRIPTION
**Pull request by Ranjansingh41**
https://github.com/Shippable/pm/issues/10962
https://github.com/Shippable/pm/issues/10961

image

### CODE REVIEW

File: sources/platform/runtime/machine-image/services-versions.md [hunk 1]
```diff
@@ -150,8 +150,7 @@ This page lists the versions of Services available for each supported [OS](/plat
 
 | Version  |  OS    | Machine Images
 |----------|---------|-------
-|3.11.1| - Ubuntu 16.04(aarch 64)|[v6.4.4](/platform/runtime/machine-image/ami-v644/)
-|3.11.0 |- CentOS 7<br>- Ubuntu 14.04<br>- Ubuntu 16.04  |[v6.5.4](/platform/runtime/machine-image/ami-v654/)
```
Comment (shrivara) [inline sources/platform/runtime/machine-image/services-versions.md:154]: Why is this removed?

Comment (Ranjansingh41) [reply sources/platform/runtime/machine-image/services-versions.md:154]: this was incorrectly updated last time

**Review by shrivara — APPROVED** [review]
